### PR TITLE
Feat withdraw wallet

### PR DESCRIPTION
**Pull request by quanace11**

### CODE REVIEW

File: apps/chat/src/app/app.tsx [hunk 1]
```diff
@@ -94,11 +95,21 @@ function AppWrapper() {
...
-		<GoogleOAuthProvider clientId={process.env.NX_CHAT_APP_GOOGLE_CLIENT_ID as string}>
-			<MezonContextProvider mezon={mezon} connect={true}>
-				<App />
-			</MezonContextProvider>
-		</GoogleOAuthProvider>
+		<MetaMaskProvider
+			debug={false}
+			sdkOptions={{
+				dappMetadata: {
+					name: 'Example React Dapp',
```
Comment (huylexuan44) [inline apps/chat/src/app/app.tsx:102]: rename and check isElectron

File: libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/WalletConfirm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+							Connect with one of our available wallet providers or create a new one.
+						</p>
+
+						<div className="my-5 space-y-4">
+							<div
+								onClick={handleWalletConfirm}
+								className="flex flex-1 flex-row justify-between p-3 text-base font-bold text-gray-900 rounded-lg bg-gray-50 hover:bg-gray-100 group hover:shadow dark:bg-gray-600 dark:hover:bg-gray-500 dark:text-white"
+							>
+								<div className="flex  gap-1 items-center ">
+									<svg aria-hidden="true" className="h-4" viewBox="0 0 40 38" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (huylexuan44) [inline libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/WalletConfirm/index.tsx:50]: move svg icon to : icons.tsx

File: libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/index.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	const [refreshHistory, setRefreshHistory] = useState(false);
+	const tokenInWallet = useMemo(() => {
+		try {
+			return JSON.parse(userProfile?.wallet ?? '{}').value;
+		} catch (error) {
+			console.error('Error parsing wallet JSON:', error);
+			return null;
+		}
+	}, [userProfile?.wallet, refreshHistory]);
+	const userId = useMemo(() => userProfile?.user?.id, [userProfile]);
```
Comment (huylexuan44) [inline libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/index.tsx:27]: remove usememo get when get userId

File: package.json [hunk 4]
```diff
@@ -37,9 +37,12 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.24.4",
+    "@gorhom/bottom-sheet": "^5.0.5",
     "@hookform/resolvers": "^3.3.4",
     "@loadable/component": "^5.16.3",
+    "@metamask/sdk-react": "^0.30.2",
     "@react-native-async-storage/async-storage": "^1.23.1",
+    "@react-navigation/native": "^7.0.0",
```
Comment (huylexuan44) [inline package.json:45]: remove

File: package.json [hunk 5]
```diff
@@ -94,6 +98,7 @@
     "react-native-device-info": "^11.1.0",
     "react-native-fs": "^2.20.0",
     "react-native-mmkv": "^2.12.2",
+    "react-native-toast-message": "^2.2.1",
```
Comment (huylexuan44) [inline package.json:101]: remove

File: package.json [hunk 6]
```diff
@@ -37,9 +37,12 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.24.4",
+    "@gorhom/bottom-sheet": "^5.0.5",
```
Comment (huylexuan44) [inline package.json:40]: remove

File: package.json (hunk 4 above)
Comment (quanace11) [reply package.json:45]: done

File: package.json (hunk 5 above)
Comment (quanace11) [reply package.json:101]: done

File: package.json (hunk 6 above)
Comment (quanace11) [reply package.json:40]: done

File: libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/index.tsx (hunk 3 above)
Comment (quanace11) [reply libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/index.tsx:27]: done

File: libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/WalletConfirm/index.tsx (hunk 2 above)
Comment (quanace11) [reply libs/components/src/lib/components/SettingProfile/SettingRightWithdraw/WalletConfirm/index.tsx:50]: done

**Pull request merged**